tests/quantization/test_bitsandbytes.py

```diff
@@ -0,0 +1,57 @@
+"""Tests whether bitsandbytes computation is enabled correctly.
+
+Run `pytest tests/quantization/test_bitsandbytes.py`.
+"""
+import pytest
+import torch
+
+from vllm.model_executor.layers.quantization import QUANTIZATION_METHODS
+
+capability = torch.cuda.get_device_capability()
+capability = capability[0] * 10 + capability[1]
+
+
+@pytest.mark.skipif(
+    capability < QUANTIZATION_METHODS["bitsandbytes"].get_min_capability(),
+    reason="bitsandbytes is not supported on this GPU type.")
+def test_load_bnb_model(vllm_runner) -> None:
```

Comment:
can we also add an e2e test that will compare the output of some model to predefined outputs?

Reply:
sure, added.